addons/point_of_sale/static/src/app/screens/ticket_screen/ticket_screen.js

```diff
@@ -423,10 +423,10 @@ export class TicketScreen extends Component {
 
         const sortOrders = (orders, ascending = false) =>
             orders.sort((a, b) => {
-                const dateA = a.date_order;
-                const dateB = b.date_order;
+                const dateA = a.date_order.ts;
+                const dateB = b.date_order.ts;
 
-                if (a.date_order !== b.date_order) {
+                if (dateA !== dateB) {
```

Comment:
```suggestion
                if (dateA.isSame(dateB)) {
```

You can use Luxon API or remove the strict equality.